device-types/Cisco/AIR-AP2802I-Q-K9.yaml

```diff
@@ -0,0 +1,14 @@
+---
+manufacturer: Cisco
+model: Aironet 2802I-Q
+slug: cisco-air-ap2802i-q-k9
+part_number: AIR-AP2802I-Q-K9
+u_height: 0
+is_full_depth: false
```

Comment:
Weight is available, please include weight

Reply:
Will be added on next commit